### DISCUSSION TITLE
Redirected to /authenticate when hitting non app route

### DISCUSSION THREAD

**Discussion by d-shannon** [Kyon147/laravel-shopify#352] [Q&A] [answered]
Hello all,

I was wondering if someone could help me pointpoint an issue.

I have a number of routes outside of my Shopify app routes that can be accessed external to Shopify e.g. "/dashboard"

The dashboard routes do not have "verify.shopify" middleware applied to them.

Ocassionaly, when I try to hit the /dashboard route, i get redirected to "/authenticate". If I clear the browser history I can hit the dashboard route as expected.

Any ideas?

Thanks!

**Comment by Kyon147** [comment]
I have a couple routes outside of the verify on some apps and they've never hit the authenticate route.

Do those routes have anything in controllers that might be causing the redirect?

**Reply by d-shannon** [reply, marked as answer]
I actually just figured this out! I created a custom auth guard which was not quite right. Nothing to do with the package. Thanks!

**Comment by hectnandez** [comment]
I made a fresh install, I followed the installation guide but I got the following error:

<img width="1455" alt="Screenshot 2024-10-09 at 18 17 38" src="https://github.com/user-attachments/assets/951f3c81-0801-4ffe-81b1-509cef00f953">

The URL's of the app is secured and the credentials already on the app were generated by the shopify-partners section, I have no Idea how to go trough here.

Thank you for the help in advance guys.
